packages/sql-mapper/test/types/mapper.test-d.ts

```diff
@@ -136,4 +136,5 @@ expectType<ErrorWithTwoParams>(errors.ParamLimitNotAllowedError)
 expectType<ErrorWithOneParam>(errors.ParamLimitMustBeNotNegativeError)
 expectType<ErrorWithOneParam>(errors.MissingValueForPrimaryKeyError)
 expectType<ErrorWithNoParams>(errors.SQLiteOnlySupportsAutoIncrementOnOneColumnError)
-
+expectType<ErrorWithNoParams>(errors.IgnoredTableNotFound)
+expectType<ErrorWithNoParams>(errors.IgnoredColumnNotFound)
```

Comment:
These is wrong, if we change the type to describe the Error params. Should be `ErrorWithTwoParams`